components/script/dom/htmlinputelement.rs

```diff
@@ -1104,6 +1097,11 @@ impl HTMLInputElement {
             _ => ()
         }
     }
+
+    #[allow(unrooted_must_root)]
```

Comment:
I'm fairly sure this is "wrong", but I don't know the right way to do it (even after [reading this](https://github.com/servo/servo/blob/master/components/script/docs/JS-Servos-only-GC.md)) so I could do with some advice here.